src/templates/layouts/pressReleaseListing/index.tsx

```diff
@@ -27,17 +27,20 @@ interface customWindow extends Window {
 }
 declare const window: customWindow
 
-export function PressReleaseListing({
-  id,
-  title,
-  introText,
-  releases,
-  menu,
-  currentPage,
-  totalPages,
-  lovellVariant,
-  lovellSwitchPath,
-}: LovellStaticPropsResource<FormattedPressReleaseListing>) {
+export function PressReleaseListing(
+  props: LovellStaticPropsResource<FormattedPressReleaseListing>
+) {
+  const {
+    id,
+    title,
+    introText,
+    menu,
+    currentPage,
+    totalPages,
+    lovellVariant,
+    lovellSwitchPath,
+  } = props
+  const releases = props['news-releases']
```

Comment:
This is where using kebab case gets a little clunky. If you know of a cleaner way to do this, please teach me! 🙏

Reply:
🤔 I guess I could do `{ id, etc, ...otherProps }` -> `const releases = otherProps['news-releases']` 🤷

Reply:
🤷